addons/im_livechat/static/src/models/public_livechat_global.js

```diff
@@ -48,25 +48,6 @@ registerModel({
         _computeChannelId() {
             return this.options.channel_id;
         },
-        /**
-         * @private
-         * @returns {FieldCommand}
-         */
-        _computeChatbot() {
-            if (this.isTestChatbot) {
-                return { data: this.testChatbotData.chatbot };
-            }
-            if (this.chatbotState === 'init') {
-                return { data: this.rule.chatbot };
-            }
-            if (this.chatbotState === 'welcome') {
-                return { data: this.livechatInit.rule.chatbot };
-            }
-            if (this.chatbotState === 'restore_session' && this.localStorageChatbotState) {
-                return { data: this.localStorageChatbotState._chatbot };
-            }
-            return clear();
```

Comment:
This case never happened before? Otherwise current diff would create the chatbot record even when not necessary.

Reply:
I don't think this happens.
I saw no guard on `.chatbot`, so I assumed I have freedom to change lifetime of model `Chatbot`, which is indeed what I did and made `Chatbot` always available.

Making `Chatbot` always available allow moving most chatbot concerns to this `Chatbot` model. Actually this `Chatbot` models the "chatbot" feature.